src/utils/nodes/BinaryExpression.ts

```diff
@@ -0,0 +1,31 @@
+import { NodeClass, NodeClassConstructor } from './Node.abstract';
+
+export class BinaryExpressionClass extends NodeClass {
+	constructor(params: NodeClassConstructor) {
+		super(params);
+	}
+
+	serialize = () => {
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		const binaryExpression = this.node as any;
+		const left = binaryExpression.left.value;
+		const right = binaryExpression.right.value;
+		const operator = binaryExpression.operator;
+		switch (operator) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously. -->
🎯 suggestion:
I suggest to handle all possible operators, you can check types in acron:
`export type BinaryOperator = "==" | "!=" | "===" | "!==" | "<" | "<=" | ">" | ">=" | "<<" | ">>" | ">>>" | "+" | "-" | "*" | "/" | "%" | "|" | "^" | "&" | "in" | "instanceof" | "**"`